### PR TITLE
Allow decimals on stocks

### PR DESCRIPTION
**Pull request by tulimaki**
- Allow updating stock to zero
- Allow decimal places for stock
- Avoid scientific notation for price and stock

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Enter here all the changes made to the development version
 
+- Allow updating stock to zero
```
Comment (ruohola) [inline CHANGELOG.md:11]: ### Changed

File: shuup_product_variations/admin/views/serializers.py [hunk 2]
```diff
@@ -79,7 +79,7 @@ class ProductCombinationSerializer(serializers.Serializer):
     combination = serializers.DictField()
     sku = serializers.CharField()
     price = FormattedDecimalField(required=False)
-    stock_count = serializers.IntegerField(required=False)
+    stock_count = serializers.DecimalField(max_digits=36, decimal_places=9, required=False)
```
Comment (ruohola) [inline shuup_product_variations/admin/views/serializers.py:82]: Would it make sense to import and use `FORMATTED_DECIMAL_FIELD_DECIMAL_PLACES` and `FORMATTED_DECIMAL_FIELD_MAX_DIGITS` here?

Comment (tulimaki) [reply shuup_product_variations/admin/views/serializers.py:82]: Might make sense but does not do much. Sounds pretty unneccessary.

Comment (ruohola) [reply shuup_product_variations/admin/views/serializers.py:82]: 🤷‍♂️ these just give off huge magic number vibes :D

**Pull request merged**